proto/firecracker.proto

```diff
@@ -92,6 +92,17 @@ message GetVMMetadataResponse {
     string Metadata = 1;
 }
 
+message DescribeInstanceInfoRequest {
```

Comment:
I prefer to not expose the API to customers right now. Removing/deprecating APIs is hard and we should do that once we really know the use-case. We can fix #492 by calling DescribeInstance by ourselves.

Reply:
Not sure what you meant by "calling DescribeInstance by ourselves".  Do you mean we should internally call the function from StopVM call?

Reply:
Anyway, implemented the fix for StopVM without needing to add another API in firecracker containerd. This should serve the purpose for now.